### DISCUSSION TITLE
TLS 不清楚如何正确配置

### DISCUSSION THREAD

**Discussion by VSfz3kkDwtjm4LAD2R24** [v2fly/v2ray-core#948] [Q&A] [answered]
你好，我正在使用 V2Ray 4.37.3。根据 https://guide.v2fly.org/#%E5%A3%B0%E6%98%8E 白话文指南，我在进行 TLS 的配置，通过 systemctl 总是启动失败，提示 21951 (code=exited, status=23)。前期生成密钥以及移动似乎都没有问题，也可以看到 /etc/v2ray 下面的两个文件打开的确是正常的密钥格式。运行 -test -config 也提示 Configuration OK。

如果服务端不加 TLS 的配置是完全没有任何问题连接使用的。

因为我不是很懂这方面的知识，配置都是按照白话文的介绍为主的。以下是我的服务端的配置，如果有谁愿意帮助我会非常感激，谢谢！

{
  "log": {
    "loglevel": "warning",
    "access": "/var/log/v2ray/access.log",
    "error": "/var/log/v2ray/error.log",
  },
  "inbounds": [
    {
      "port": *,
      "protocol": "vmess",
      "settings": {
        "clients": [
          {
            "id": "*",
              "alterId": 64
          }
        ]
      },
      "streamSettings": {
        "network": "tcp",
        "security": "tls",
        "tlsSettings": {
          "certificates": [
            {
              "certificateFile": "/usr/local/etc/v2ray/v2ray.crt",
              "keyFile": "/usr/local/etc/v2ray/v2ray.key"
            }
          ]
        }
      } 
    }
  ],
  "outbounds": [
    {
      "protocol": "freedom",
      "settings": {}
    }
  ]
}

**Comment by okudayukiko** [comment, marked as answer]
注意檔案權限。

**Reply by VSfz3kkDwtjm4LAD2R24** [reply]
感谢您的回复。请问您指的档案权限是指 V2Ray 没有权限读取 /usr/local/etc/ 下的密钥文件吗？

**Reply by VSfz3kkDwtjm4LAD2R24** [reply]
我明白了！！！！！！！对不起我没有好好查看 chmod 相关的文档。实在是非常感谢您的指点！
